### PR TITLE
Feat responsive

### PR DESCRIPTION
**Pull request by JowelTisso**
* This is PR is for resposiveness

### CODE REVIEW

File: src/src/pages/videos/videoPlayer/VideoPlayer.css [hunk 1]
```diff
@@ -24,7 +24,64 @@
...
+
+@media screen and (max-width: 480px) {
+  .video-playlist .video-card-link {
+    width: 50%;
+  }
+
+  .video-frame {
+    height: 240px;
+    width: 100%;
+  }
```
Comment (manismk) [inline src/src/pages/videos/videoPlayer/VideoPlayer.css:77]: try to use rems here too

**Review by manismk — APPROVED** [review]
Looks good.
Responsiveness works properly

**Review by shristabaruah — APPROVED** [review]
Screenshot_2022-06-11-23-02-30-536_com android chrome
- suggestion: You can add some gap between the videos.. Other than that it looks great!

File: src/src/pages/videos/videoPlayer/VideoPlayer.css (hunk 1 above)
Comment (JowelTisso) [reply src/src/pages/videos/videoPlayer/VideoPlayer.css:77]: ok

**Comment by JowelTisso** [comment]
> * suggestion: You can add some gap between the videos.. Other than that it looks great!

Ok, I guess I missed for this screen size.

**Pull request merged**